libs/shared/src/containers/Login/index.tsx

```diff
@@ -109,6 +109,9 @@ export const Login = (props: Props) => {
       setPercentComplete(-1);
       if (!__cancel) {
         setValidateFn(() => badPwdValid(password!));
+        // Force validation update by toggling forceValidate
+        setForceValidate(v => !v);
+        setForceValidate(v => !v);
```

Comment:
Because onValidate is no longer a trigger for re-validation, we must manually force the validation if we change the validate fn.

Technically, this is only necessary if forceValidate is true, (it is always true here)